schemas/condition.schema.json

```diff
@@ -64,6 +64,18 @@
             {
               "const": "URL_TYPE",
               "title": "Check if the site is on subdirectory or subdomain integration"
+            },
+            {
+              "const": "INTEGRATION",
```

Comment:
I don't think it's so much "Check the integration of the project" as "Specify which integration should use this definition". The reason is that a Connect project will also need the JS settings, but they should only be loaded by the JS library. And also a JS project can pass via Connect, but we don't want to load the JS settings in that case.

So in `Condition.ts` in Connect, `definitionChecker` should have
```
if (type === "INTEGRATION" && payload !== "connect") {
        return false;
      }
```

Reply:
Aussi je pense qu'on peut peut-être spécifier que `payload` est un Array de strings, comme ça on peut facilement avoir un payload ajouté sur les 2.